app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -970,7 +970,7 @@ public function applySidebarData($data)
                 $item = $this->getCustomerCart()->getItemById($itemId);
                 if ($item) {
                     $this->moveQuoteItem($item, 'order', $qty);
-                    $this->removeItem($itemId, 'cart');
+                    $data['remove'][$itemId] = 'cart';
```

Comment:
What if `$data['remove']` does not exist?

Reply:
In this case we put value under key that will be processed on line 996. I have rewritten that code to be more readable.